.github/workflows/ci.yml

```diff
@@ -7,7 +7,7 @@ env:
 on:
   pull_request:
   push:
-    branches: ['master', 'zio-2']
+    branches: ['master', 'series/2.x']
```

Comment:
I think can just do `series/2x` at this point.

Reply:
Theoretically we could want to publish something in the `series/1.x` branch, right?

Reply:
I don't mind if we don't do that :shrug: :+1: